mantle/kola/tests/misc/secex.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package misc
+
+import (
+	"github.com/coreos/coreos-assembler/mantle/kola/cluster"
+	"github.com/coreos/coreos-assembler/mantle/kola/register"
+	"github.com/coreos/coreos-assembler/mantle/platform"
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Run:             verifySecex,
+		ClusterSize:     1,
+		Name:            "coreos.secex.ensure",
+		Flags:           []register.Flag{},
+		Tags:            []string{"secex"},
+		Architectures:   []string{"s390x"},
+		Platforms:       []string{"qemu-unpriv"},
+		MinMemory:       4096,
+		SecureExecution: true,
```

Comment:
Now that we'll specify `--qemu-secex`, I think we can drop this knob too and in turn we should be able to make this test an external test instead. WDYT?

Reply:
yep, sound good. than i'll drop here 2nd commit